templates/user/auth/captcha.tmpl

```diff
@@ -0,0 +1,24 @@
+{{if .EnableCaptcha}}{{if eq .CaptchaType "image"}}
+	<div class="inline field">
+		<label></label>
```

Comment:
You mean empty `<label></label>`?  I don't think it can be removed. 

<details>

![](https://user-images.githubusercontent.com/76462613/201447397-1eea1dbb-143d-4d8f-aea2-6ba467e62591.jpeg)

</details>

Reply:
Perhaps we should add a comment then…

Reply:
Done. Add a comment.